res/css/views/rooms/_EventTile.scss

```diff
@@ -152,7 +153,6 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
         display: inline-block;
         padding-bottom: 0px;
         padding-top: 0px;
-        max-width: calc(100% - $left-gutter);
```

Comment:
|Before|After|
|---------|------|
|![before2](https://user-images.githubusercontent.com/3362943/174442813-077ae7bb-98eb-4960-bca2-93b80fc16cca.png)|![after2](https://user-images.githubusercontent.com/3362943/174442809-af177eab-83ce-4f47-aa5d-5ea1974e7ab5.png)|

Because `max-width: calc(100% - $left-gutter)` is no longer applied to  `mx_DisambiguatedProfile` inside reply tiles.